### DISCUSSION TITLE
6.6y的内核无法启动吗？

### DISCUSSION THREAD

**Discussion by Lemon1151** [ophub/amlogic-s9xxx-armbian#2041] [Rockchip]
![image](https://github.com/ophub/amlogic-s9xxx-armbian/assets/149338935/b98eae7c-fd99-4fcc-af50-3dcec6abbeeb)

**Comment by Lemon1151** [comment]
> ![image](https://github.com/ophub/amlogic-s9xxx-armbian/assets/149338935/b98eae7c-fd99-4fcc-af50-3dcec6abbeeb)

提示找不到根目录

**Reply by chouex** [reply]
找不到UUID, 你的ROOT block的UUID正確嗎?
`# ls -la /dev/disk/by-uuid/`

**Comment by 13584452567** [comment]
详细的日志截图，如果有ttl日志最好，没有的话终端截图也行。
设备无法启动的问题最好去issues那边发
